ang/civicase-base/directives/checkbox.directive.js

```diff
@@ -0,0 +1,65 @@
+(function (angular) {
+  var module = angular.module('civicase-base');
+
+  module.directive('civicaseCheckbox', function () {
+    return {
+      restrict: 'EA',
+      controller: 'civicaseCheckboxController',
+      templateUrl: '~/civicase-base/directives/checkbox.directive.html',
+      transclude: true,
```

Comment:
Why do we need transclude here?

Reply:
It's for the label. We have two options:

1 - to transclude the content of the directive and use it for the label: `<checkbox>label</checkbox</checkbox>`.
2 - use an attribute as the label: `<checkbox label="label"></checkbox>`

Went with option 1 because it supports labels with markup and it's the less verbose out of the two options.

Should we use option 2 instead?